### ISSUE TITLE
`getUnderlyingDataAsync()` and `getSummaryData()` returns all columns as `string`

### ISSUE THREAD

**Issue by gyulalaszlo** [tableau/extensions-api#41]
When using `getUnderlyingDataAsync` in Tableau Desktop Beta build `17.0929.1346` the returned columns are always reported as having `string` dataType.

When testing with Superstore, calling (example code)

```js
worksheet
    .getUnderlyingDataAsync({
        maxRows          : 0,
        includeAllColumns: true
     }).then( table => table.columns.forEach(c=>
         addToLog(`${c.name} : ${c.dataType}`)
     ));
```

results in:

```
...
Category : string
Order Date : string
Order Profitable? : string
...
```

Other Tableau workbooks also exhibit the same behaviour, as have previous versions of frelard.

**Comment by benlower** [comment]
@gyulalaszlo @tfoldi thanks for reporting this we will take a look

**Comment by lbrendanl** [comment]
Yeap, this is definitely our bug, was able to reproduce with any sample.

**Comment by tfoldi** [comment]
any update guys?

**Comment by elatt** [comment]
This appears to still be present in build 2018.2 Beta 1

**Comment by johnDance** [comment]
This has been fixed internally, and will be part of our our next release. Thanks!

**Comment by marco79cgn** [comment]
Has this already been released? If so, which Tableau version contains the bugfix?

**Comment by johnDance** [comment]
Hi, this should be fixed in the final 2018.2 and all releases going forward.

**Comment by KeshiaRose** [comment]
Closing this as it was fixed in 2018.2. This is different from the bug with values being returned as strings which we will keep track of in [#192](https://github.com/tableau/extensions-api/issues/192).
